.github/workflows/build-iso.yml

```
@@ -69,19 +69,25 @@
           -H "Accept: application/vnd.github.v3+json" \
           "https://api.github.com/repos/${{ github.repository }}/actions/artifacts" \
-          | jq '.artifacts[] | select(.name=="felux-v${{ steps.iso_version.outputs.version }}") | .id')
+          | jq '.artifacts[] | select(.name=="${{ steps.build.outputs.iso_name }}") | .id')
 
-        ARTIFACT_URL="https://github.com/${{ github.repository }}/actions/artifacts/$ARTIFACT_ID/felux-v${{ steps.iso_version.outputs.version }}"
+        ARTIFACT_URL="https://github.com/${{ github.repository }}/actions/artifacts/$ARTIFACT_ID/${{ steps.build.outputs.iso_name }}"
         echo "artifact_url=$ARTIFACT_URL" >> $GITHUB_OUTPUT
 
+    - name: Format Build Date
+      id: format_date
+      run: echo "formatted_date=$(date -d '${{ steps.iso_build_date.outputs.date }}' +"%Y/%m/%d")" >> $GITHUB_ENV
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:92: Double quote to prevent globbing and word splitting

(shellcheck)